ehr/reconcile/planner.go

```diff
@@ -0,0 +1,87 @@
+package reconcile
+
+import (
+	"context"
+
+	duration "github.com/xhit/go-str2duration/v2"
```

Comment:
I'm not a fan of taking a dependency for this. We support literally four values, I don't think we need a 3rd-party dependency to handle that now, or in the future.

Reply:
I removed the dependency. I don't want to have to update 2 services if we decide to support 60 day reports one day, so I had to implement the parsing logic myself. I think this results in less readable code.